functional-tests/devscripts/route_tests.sh

```diff
@@ -46,7 +46,7 @@ function checkPrereq {
 		
 	if $SEND_TO_ZABBIX; then
 		rpm -ivh https://repo.zabbix.com/zabbix/3.0/rhel/7/x86_64/zabbix-release-3.0-1.el7.noarch.rpm
-		yum install -y zabbix-sender
+		yum install -y --nogpgcheck zabbix-sender
```

Comment:
could you clarify what is the fix is about in the description

Reply:
There is an issue, where `zabbix-sender` is complaining that the package's GPG key is not installed. This broke our route tests. I'm adding the `--nogpgcheck` flag to YUM to avoid this problem. It's been broken with the recent versions of zabbix-sender.